src/app.js

```diff
@@ -265,7 +265,7 @@ export default function(app) {
           removeElement(element, reusableChild[0], reusableNode)
         }
       }
-    } else if (node !== oldNode) {
+    } else if (node != oldNode) {
```

Comment:
If it's come to this point, we're talking about two exact same objects (with rfs unchanged), or two same text nodes (strings). Either way, I guess it _can_ fail if we have something like

```
<div>[object Object]</div>
```

turning into

```
<div><div></div></div>
```

So I guess this should be remain `!==` actually.